api/src/main/java/au/org/aodn/nrmn/restapi/controller/IngestionController.java

```diff
@@ -25,84 +26,157 @@
 import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import io.swagger.v3.oas.annotations.tags.Tag;
 
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.*;
+
 @RestController
 @RequestMapping(path = "/api/v1/ingestion")
 @Tag(name = "Survey Ingestion")
 public class IngestionController {
 
+    protected static final Logger logger = LoggerFactory.getLogger(IngestionController.class);
+
+    @Value("${nrmn.jobs.scheduleInitDelay:1000}")
+    protected int scheduleInitDelay;
+
+    @Autowired
+    protected StagedJobRepository jobRepository;
+
     @Autowired
-    StagedJobRepository jobRepository;
+    protected StagedRowRepository rowRepository;
 
     @Autowired
-    StagedRowRepository rowRepository;
+    protected SurveyIngestionService surveyIngestionService;
 
     @Autowired
-    SurveyIngestionService surveyIngestionService;
+    protected StagedJobLogRepository stagedJobLogRepository;
 
     @Autowired
-    StagedJobLogRepository stagedJobLogRepository;
+    protected UserActionAuditRepository userActionAuditRepository;
 
     @Autowired
-    UserActionAuditRepository userActionAuditRepository;
+    protected SpeciesFormattingService speciesFormatting;
 
     @Autowired
-    private SpeciesFormattingService speciesFormatting;
+    protected MaterializedViewService materializedViewService;
 
     @Autowired
-    private MaterializedViewService materializedViewService;
+    protected GlobalLockService globalLockService;
```

Comment:
are there specific reasons that you changed to `protected`? I don't see where else other than the `IngestionController` class these variables are being used.